Code/Geometry/Transform2D.cpp

```diff
@@ -18,20 +18,19 @@
 namespace RDGeom {
 
 void Transform2D::setToIdentity() {
-  unsigned int i, id;
   double *data = d_data.get();
   memset(static_cast<void *>(data), 0, d_dataSize * sizeof(double));
-  for (i = 0; i < DIM_2D; i++) {
+  unsigned int id;
```

Comment:
This doesn't need to be declared outside the loop; it could be just defined on line 25.